provers/sp1/guest-cl-agg/src/main.rs

```diff
@@ -10,7 +10,10 @@ fn main() {
         "At least one CL proof is required for aggregation"
     );
 
-    let cl_proof_pp_start = read_and_validate_next_proof();
+    // TODO: Do this in better
+    let mut cl_proof_pp_start = read_and_validate_next_proof();
+    cl_proof_pp_start.initial_snapshot.slot += 1;
+
```

Comment:
Need to add context why `cl_proof_pp_start.initial_snapshot.slot += 1` was done.